python/activator/middleware_interface.py

```diff
@@ -64,6 +64,8 @@
 # VALIDITY-HACK: local-only calib collections break if calibs are not requested
 # in chronological order. Turn off for large development runs.
 cache_calibs = bool(int(os.environ.get("DEBUG_CACHE_CALIBS", '1')))
+# TODO: workaround for DM-40193
+cache_anything = bool(int(os.environ.get("DEBUG_CACHE_INPUTS", '1')))
```

Comment:
The transitions from Helm values `true`/`false` to K8s YAML `1`/`0` to Python `True`/`False` are a little mind-bending, but I guess it's better than getting `true` and `True` confused.

Reply:
Unfortunately, since environment variables must be strings, and any nonempty string casts to `bool` as `True`, there needs to be an explicit translation step _somewhere_. I picked this convention mainly because it mirrors the existing `LSST_DISABLE_BUCKET_VALIDATION`.